modin/core/execution/ray/generic/io/io.py

```diff
@@ -134,56 +148,51 @@ def func(df, **kw):
                 Arguments to pass to ``pandas.to_csv(**kw)`` plus an extra argument
                 `partition_idx` serving as chunk index to maintain rows order.
             """
-            if kw["partition_idx"] != 0:
+            idx = kw["partition_idx"]
```

Comment:
```suggestion
            partition_idx = kw["partition_idx"]
```
for the sake of better understanding